### PR TITLE
Fix #3697 legacy architecture note

### LINKED ISSUE #3697: Feedback on page: /guides/v2.3/cloud/architecture/pro-architecture.html
"For Pro projects created before October 23, 2017, the architecture is slightly different."

This statement is not strictly correct as older projects are being upgraded to the new approach.

We should say something like:

"Pro projects created before October 23, 2017, were originally provisioned using a slight different architecture.  To be upgraded, please contact your Customer Success Manager."

### PR DESCRIPTION
**Pull request by hguthrie**
Updated a repeating note about the legacy architecture, per request in GitHub issue 3697.
Corrected an instance where submit a ticket phrase was not entirely appropriate.

The note appears in the [Pro Architecture topic](https://devdocs.magento.com/guides/v2.2/cloud/architecture/pro-architecture.html).

### CODE REVIEW

File: guides/v2.1/cloud/trouble/pro-env-management.md [hunk 1]
```diff
@@ -7,8 +7,9 @@ Previously, you accessed Pro Staging and Production environments by using an SSH
 
 To add these environments to the Project Web Interface, review this entire document, complete a few preparatory steps, and submit a ticket. Your ticket is added to a queue for updating _existing_ Pro projects. The process may take time to complete, so check your ticket for details, timing, and other important information.
 
-{% include note.html type="info" content="The Staging and Production environments for **New projects provisioned October 23, 2017 and later** are in the Project Web Interface. You must submit a ticket to convert a project created before this date." %}
-
+{:.bs-callout .bs-callout-info}
+The Staging and Production environments for **New projects provisioned October 23, 2017 and later** are in the Project Web Interface. You must contact your Customer Success Manager to convert a project created before this date.
```
Comment (meker12) [inline guides/v2.1/cloud/trouble/pro-env-management.md]: Not sure what "in the Project Web Interface" means--Managed from?
Project Web Interface ==> Project Web UI    Would it be helpful to link Project Web UI to the place in the docs that discusses it?

Comment (hguthrie) [reply guides/v2.1/cloud/trouble/pro-env-management.md]: Project Web UI is a standard phrase used throughout the Cloud documentation to indicate the web-accessible user interface for a Cloud project. I'm not sure this is the place for that.

Comment (hguthrie) [reply guides/v2.1/cloud/trouble/pro-env-management.md]: All I did here was change to the appropriate note style instead of the include.

**Comment by hguthrie** [comment]
running tests

**Pull request merged**